sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -68,6 +70,8 @@ export interface TracingClient {
         span: TracingSpan;
         updatedOptions: OptionsWithTracingContext<Options>;
     };
+    trace<Arguments, Return>(name: string, args: Arguments, methodToTrace: () => Return, onStartTracing?: (span: TracingSpan, args: Arguments) => void, onEndTracing?: (span: TracingSpan, args: Arguments, rt?: Return, error?: unknown) => void, options?: OperationTracingOptions, spanKind?: TracingSpanKind): Return;
```

Comment:
Same here, we will not be able to add this to our core tracing APIs

Reply:
I can move this to inference or other more public facing library and let inference consume it.